test/sanity/issue5242-nwjs-app-open-file/test.py

```diff
@@ -0,0 +1,61 @@
+import time
+import os
+import sys
+import shutil
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from nw_util import *
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+testdir = os.path.dirname(os.environ["CHROMEDRIVER"])
```

Comment:
The dir where chromedriver resides should not be used as the temp directory of the test case.

Reply:
Updated